### ISSUE TITLE
Question about formatting of negative values in IDV.

### ISSUE THREAD

**Issue by bolazec** [mflorence99/weighted-numbers#1]
In this code I expected the formatted text to contain negative values.
```ts
describe('Negative IDV', () => {
  const idv = new IDV({ gallons: -2, quarts: -3, pints: 1 });

  test('Values have been normalized', () => {
    expect(idv.pints).toEqual(-1);
    expect(idv.quarts).toEqual(-2);
    expect(idv.gallons).toEqual(-0);
    expect(idv.pecks).toEqual(-1);
    expect(idv.bushels).toEqual(0);
    expect(idv.chaldrons).toEqual(0);
  });

  test('IDV is formatted as readable English text', () => {
    expect(idv.format()).toEqual('(1 pecks, 2 quarts, 1 pints)');
  });

});
```

**Comment by mflorence99** [comment]
Good call! I didn't think `-1 pecks, -2 quarts, -1 pints` was very readable and certainly `£-1 -10s -6d` doesn't look good. So I decided to enclose negative values in parentheses (that's easy to miss in the `expect(...)` statement) and show the absolute values of each unit. So `(£1 10s 6d)` instead.

If this were a real library, you'd want to provide a good deal more control over formatting -- localization, for one, maybe abbreviations like `1gal 2qt` for another. But I decided to stop when the code was as functionally complete as I could make it, as its purpose is really academic or didactic.
